### DISCUSSION TITLE
SuperCollider -> Hangouts

### DISCUSSION THREAD

**Discussion by KonVas** [ExistentialAudio/BlackHole#82] [General]
System: MacOS(10.15.3 Catalina).
I am trying to use Hangouts for testing reasons, routing SuperCollider(fig1) to Hangouts(fig2) for example. Although I can see SC used the out that I am selecting succesfully the sound is not ported in the hangouts when I am selecting the hangouts:mic option set to BlackHole, similarly I tried everything I read on this git's issues including using aggregate and or multiout devices configs but no success. Settings attached (fig3).
![Image](https://user-images.githubusercontent.com/6897992/77665993-73163e00-6f91-11ea-9c5e-76a152346456.png)
Fig 1.

![Image](https://user-images.githubusercontent.com/6897992/77665862-44986300-6f91-11ea-8d6b-e029289bf239.png)
Fig 2.

![Image](https://user-images.githubusercontent.com/6897992/77666596-33038b00-6f92-11ea-9e4c-4096708c53d5.png)
Fig 3

**Comment by devinroth** [comment]
No idea why it isn’t working off the top of my head. I’ll download SuperCollider and try it out. 

Devin

**Comment by berkeleybernie** [comment]
I'm new to this and was trying to set this up using Reaper DAW as a mixer to route output to OBS using an aggregate device of Blackhole/Audio Interface(Zoom H5) in Reaper. I was getting no audio in OBS. Turned out that one has to configure the routing in Reaper so that the aggregate device doesn't output back to the input device (in my case, I was using a Zoom H5). Once I used the routing matrix (not in the audio preferences; an additional step) in Reaper to include Blackmagic channels, OBS was able to see the Reapers audio output.

Maybe some kind of routing necessary in SuperCollider as well?

**Comment by KonVas** [comment]
> Maybe some kind of routing necessary in SuperCollider as well?

If I am getting it right, you want SC to use BlackHole as output, assuming yes: evaluate this block as one piece of code:

```js
(
// s = Server.local;

//using the on-board mic, unnecessary here but handy
s.options.inDevice = "Built-in Microph";

//must match to the booting info for the audio drivers provided as strings in post window. 
s.options.outDevice = "BlackHole 16ch";

// new settings will take effect after rebooting the server.
s.reboot; 
)
```
